trillian_map_api.proto

```diff
@@ -137,6 +137,25 @@ message SetMapLeavesResponse {
   SignedMapRoot map_root = 2;
 }
 
+message WriteMapLeavesRequest {
+  int64 map_id = 1;
+  // The leaves being set must have unique Index values within the request.
+  repeated MapLeaf leaves = 2;
+  bytes metadata = 3;
+  // The map revision to associate the leaves with. The request will fail if
+  // this revision already exists, does not match the current write revision, or
+  // is negative. If revision = 0 then the leaves will be written to the current
+  // write revision.
+  int64 revision = 4;
```

Comment:
Would `expect_revision` be a touch clearer to reflect the check-and-set functionality this supports?

Reply:
Good call.